src/components/Settings.js

```diff
@@ -63,6 +64,7 @@ class SettingsForm extends React.Component {
   render() {
     return (
       <form onSubmit={this.submitForm}>
+      <h3>{this.state.username}</h3>
```

Comment:
```suggestion
      <h3>{this.state.username}</h3>
```
Make it <h1> instead fo <h3>
Suggestion comments form the base repository for the code changes.